<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>1391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {
background:#aaaaaa;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="g_emp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Gestion des employes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/new/prefix1/admin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fire_">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../employe.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="g_bag">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Gestions des bagages</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../valise.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="g_voy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Gestion des voyageurs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>500</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../voyageur-avec-une-valise.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="g_avion">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Gestion des avions</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>540</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../avion.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="compte">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/new/prefix1/g_account.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="g_vol">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
text-align: left;
border: 1px solid #6a1616;
}
QPushButton:hover {
color:#fff;
background:#4cb1db ;
border: none;
text-align: left;
}
</string>
     </property>
     <property name="text">
      <string>       Gestion des Vols</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>350</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../flying.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../aa.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>300</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../home.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>1511</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#QWidget {
background:#4cb1db ;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 5px solid #4cb1db;
}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #4cb1db ;
border-bottom-color: #4cb1db ; /* same as the pane color */
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 120px;
padding: 8px;
color: #6a1616;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4cb1db , stop: 0.4 #4cb1db , stop: 0.5 #4cb1db , stop: 1.0 #4cb1db );
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>1011</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>561</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Modern No. 20</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>veuillez remplir les information suivantes </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>numero de serie  d'avion </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>kilometrage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>modele d'avion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_modele">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(189, 189, 189);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 188, 188);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="la_marque">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 188, 188);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_kil">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>420</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 189, 189);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>1001</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>671</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Modern No. 20</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>afficher les information  d'une avion</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_avion">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>101</y>
        <width>861</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_afficher">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>1011</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>561</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Modern No. 20</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Suppression une avion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>tapez le numero de serie </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_sup">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>211</y>
        <width>691</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>671</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Modern No. 20</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>modifier les information d'une avion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>identifiant avion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod_id">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod_kil">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>nouveau kilometrage</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod_modele">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod_marque">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>modele</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_mod">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>481</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Rechercher</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_13">
     <attribute name="title">
      <string>Envoyez e-mails</string>
     </attribute>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>710</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
border: 1px solid #6a1616;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background:#6a1616;
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>710</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
border: 1px solid #6a1616;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background:#6a1616;
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_14">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
